src/test/kotlin/com/atlassian/performance/tools/virtualusers/DockerJiraFormula.kt

```diff
@@ -52,7 +52,12 @@ class DockerJiraFormula(
         db: SudoSshUbuntuContainer
     ): DockerJira {
         val peerJiraPort = 8080
-        val publishedJiraPort = jira.ports.bindings[ExposedPort.tcp(peerJiraPort)]!!.single().hostPortSpec.toInt()
+        val publishedJiraPort = jira
+            .ports
+            .bindings[ExposedPort.tcp(peerJiraPort)]!!
+            .single { it.hostIp == "0.0.0.0" }
+            .hostPortSpec
+            .toInt()
         val dockerJira = DockerJira(
             hostAddress = URI("http://localhost:$publishedJiraPort/"),
             peerAddress = URI("http://${jira.peerIp}:$peerJiraPort/")
```

Comment:
Unfortunately, the inlined `lib/sshubuntu` couldn't be replaced by the actual `ssh-ubuntu` yet. It doesn't model multiple addresses yet (host vs peer or public vs private).